app/src/views/Cart.vue

```diff
@@ -254,7 +262,7 @@ function useCart() {
     if (success) clearCart();
   }
 
-  return { cart, lsCart, cartSummaryString, grantMetadata, clearCart, completeCheckout, fetchQuotes, initializeCart, clrPredictions, clrPredictionsByToken, showEquivalentContributionAmount, equivalentContributionAmount, executeCheckout, removeFromCart, setCart, status, txHash, updateCart }; // prettier-ignore
+  return { cart, lsCart, cartSummaryString, grantMetadata, clearCart, completeCheckout, fetchQuotes, initializeCart, clrPredictions, clrPredictionsByToken, showEquivalentContributionAmount, equivalentContributionAmount, executeCheckout, removeFromCart, setCart, status, txHash, updateCart, isCorrectNetwork }; // prettier-ignore
```

Comment:
This is just a stylistic personal preference of mine. When there's a long return statement or a lot of imports from one file, when it's only a little bit over the line limit, I prefer keeping it on one line instead of wrapping and taking up a lot of lines. 

This one has gotten very long so it probably should have the prettier-ignore removed so it wraps, and same with the destructuring from `useCartStore()` a few lines up.

I just pushed 86bf08a67dbb59687eb6fffbc45de0c668559d9f which removes the `prettier-ignore` from those two lines and sorts the items alphabetically so it's easier to read

Reply:
Got it. Thanks.